src/citrine/gemtables/variables.py

```diff
@@ -339,6 +339,63 @@ def __init__(self,
         self.type_selector = type_selector
 
 
+class LocalAttributeAndObject(Serializable['LocalAttributeAndObject'], Variable):
+    """[ALPHA] Attribute marked by an attribute template for the root of a material history tree.
+
+    Parameters
+    ----------
+    name: str
+        a short human-readable name to use when referencing the variable
+    headers: list[str]
+        sequence of column headers
+    template: Union[UUID, str, LinkByUID, AttributeTemplate]
+        attribute template that identifies the attribute to assign to the variable
+    object_template: Union[UUID, str, LinkByUID, AttributeTemplate]
+        attribute template that identifies the attribute to assign to the variable
+    attribute_constraints: List[Tuple[Union[UUID, str, LinkByUID, AttributeTemplate], Bounds]]
+        Optional
+        constraints on object attributes in the target object that must be satisfied. Constraints
+        are expressed as Bounds.  Attributes are expressed with links. The attribute that the
+        variable is being set to may be the target of a constraint as well.
+    type_selector: DataObjectTypeSelector
+        strategy for selecting data object types to consider when matching, defaults to PREFER_RUN
```

Comment:
Just double-checking, Here we are not mapping `attribute_template_type`, I do see it in the openapi spec, is this because we are not going to use it on the SDK?

Reply:
It is in the openapi spec, but that field is not populated by any of the existing Variable objects in the SDK.  For the life of me, I don't see what impact that field is supposed to have, based on digging through platform-backed.